config/proto/service.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package hermes;
+
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto";
+
+service Hermes {
+
+  // Start monitoring a new storage system.
+  rpc StartMonitoringStorageSystem(HermesProbeRequest) returns (HermesProbeResponse) {}
+
+  rpc StopMonitoringStorageSystem(StopMonitoringSystemRequest) returns (StopMonitoringSystemResponse) {}
+
+  // Lists the storage systems being monitored at the moment.
+  rpc ListMonitoredStorageSystems(ListMonitoredSystemsRequest) returns (ListMonitoredSystemsResponse) {}
+}
+
+message HermesProbeRequest {
+  required HermesProbeDef probe_confg = 1;
+}
+
+message HermesProbeResponse {}
+
+message StopMonitoringSystemRequest {
+ required Target target = 1;
+}
+
+message StopMonitoringSystemResponse {}
```

Comment:
You may want to add some sort of status code as a message field. But you can leave this for future work, up to you. (same for the rest of the response messages.

You could do something like what they exemplify here [1] or here [2].

[1] https://developers.google.com/protocol-buffers/docs/proto3#other
[2] https://developers.google.com/protocol-buffers/docs/proto3#any

Reply:
I have created a new issue, #16, which describes this. I think it would be better to implement this in the future when I know more about what to expect from the return.